erpnext/accounts/doctype/cashier_closing/cashier_closing.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+// License: GNU General Public License v3. See license.txt
+
+frappe.ui.form.on('Cashier Closing', {
+
+	expense: function(frm){
```

Comment:
all calculations must be on server side

https://github.com/frappe/erpnext/wiki/Coding-Standards#all-business-logic-must-be-api-friendly-and-must-be-implemented-in-server-side-python

Reply:
now the calculations are on the server side